### ISSUE TITLE
How to interpret make_viewpoints.py output

### ISSUE THREAD

**Issue by Tocci89** [nservant/HiC-Pro#136]
Hi Nservant,
I'm using the make_viewpoints.py script but I'm not sure about the output.
First of all, what is the meainig of the statistics report? (CAP-REP read pairs;CAP-CAP read pairs; REP-REP read pairs; Excluded reads; UA reads).
Then, is each track of the bedgraph file corresponding to an entry of the input bed file? And what are the integer numbers in the 4th column of each track?
Thanks in advance.

**Comment by werhoog** [comment]
yeah, thanks for your good quesiton.I also got trouble in understanding  what the function of such python script ? Is it the figure  out where are  all of other interaction regions of the viewpoint?
thanks!

**Comment by nservant** [comment]
Hi,
This script was designed to generate a 4C-like profile, mainly in the case of capture-C analysis.
Basically, for all regions (lines) in the input BED file, it will report its interactor and how many time we find an interaction (4th column). Back to a Hi-C matrix, this is like showing a line of a matrix ....

In the context of capture-C, the CAP (capture) regions correspond to the line of the BED files, and the REP (reporter) regions to its interactors. In addition, it is common to discard short range interaction in such analysis. That's why you can define an "exclusion" region around the capture viewpoint (reported as Excluded reads).

I will update the manual of this tool for the next release.
Is it clear now ?

**Comment by Tocci89** [comment]
Thanks a lot!
I figured it out but I was in doubt for some points. 
Now everything is clear.

**Comment by werhoog** [comment]
yeah,got it .
expect for your next release.

**Comment by Tocci89** [comment]
Hi Nservant,
If I don't explicitly declare the -e option, is there any default value?
In other words, what value should I give to -e if I don't want to exclude any region around my view point?

**Comment by nservant** [comment]
Hi,
By default, there is no exclusion around the viewpoint.
N
